webapp/lib/charts.dart

```diff
@@ -204,4 +204,27 @@ class DailyTimeseriesLineChartView {
     var chartConfig = new chartjs.ChartConfiguration(type: 'line', data: chartData, options: chartOptions);
     chart = chartjs.Chart(canvas.getContext('2d'), chartConfig);
   }
- }
+}
+
+class SystemMetricsTimeseriesLineChartView extends DailyTimeseriesLineChartView {
+  @override void updateChart(List<Map<DateTime, num>> updatedCountsAtTimestampList, {String timeScaleUnit = 'day', num upperLimit}) {
```

Comment:
I suggest making `upperLimit` an optional named arg directly on the method in `DailyTimeseriesLineChartView`. It would reduce code duplication, and we might want to add it to the other charts as well in the future.

Reply:
Yes. I think it's a good suggestion. We can find a use for the upperLimit there too. I've reverted it back to use the `DailyTimeseriesLineChartView`